### PR TITLE
Removed systemd

### PR DESCRIPTION
**Pull request by robbe-haesendonck**
To get the containers working unprivileged systemd was removed from the container.
The entrypoint/bash script launches the registry and redis (when the env variables match)

I've also removed rsyslogd, this could be added back if required.

### CODE REVIEW

File: config/confd/templates/logentries.conf.tmpl [hunk 1]
```diff
@@ -1,7 +0,0 @@
-{{if getenv "LOGENTRIES_TOKEN"}}
```
Comment (dfunckt) [inline config/confd/templates/logentries.conf.tmpl:1]: Support for Logentries cannot be removed as we're currently using it for production logs. Did you bump into issues with it?

I'll have to double-check that the no-systemd base image actually supports Logentries before switching to it.

Comment (dfunckt) [reply config/confd/templates/logentries.conf.tmpl:1]: Right, the base image does not care about Logentries -- all that's needed is configuring rsyslog to send logs there. So, yeah, let's please revert the rsyslog-related changes.

**Comment by ab77** [comment]
@robbe-haesendonck we still require Systemd variant of this component, as it is used in local-dev environments on top of mDNS. If you could update the PR to build both in parallel (using .resinci.yml), it should be OK to merge.

**Comment by ab77** [comment]
please re-open if still required

**Pull request closed without merging**